site/en/docs/extensions/migrating/to-service-workers/index.md

```diff
@@ -100,7 +101,16 @@ textEl.select();
 document.execCommand('copy');
 ```
 
-Communicate between offscreen documents and extension service workers using  [message passing](/docs/extensions/mv3/messaging/).
+Communicate between offscreen documents and extension service workers using [message passing](/docs/extensions/mv3/messaging/).
+
+## Convert localStorage to chrome.storage.local {: #convert-localstorage }
+
+The web platform's [`Storage`](https://developer.mozilla.org/docs/Web/API/Storage) interface (accessible from `window.localStorage`) cannot be used in a background script. You have two options for replacing it.
+
+* Move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
+* Replace it with calls to the [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace.
+
+In the latter case, you will still need to handle the conversion in an offscreen document.
```

Comment:
```suggestion
The web platform's [`Storage`](https://developer.mozilla.org/docs/Web/API/Storage) interface (accessible from `window.localStorage`) cannot be used in a background script. To address this:

1. Move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
1. (Optional) Replace it with calls to the [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace. For details, see [the instructions](/docs/extensions/reference/storage/#can-extensions-use-storage).
```

Reply:
I have the anchor this links to in my local copy. I'll push it after you OK the language.

Reply:
This looks good 👍